static/CNAME

```diff
@@ -1 +0,0 @@
-owddm.com
```

Comment:
This is rather important to have, as [without a CNAME](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site), the hosting will not work